### ISSUE TITLE
Problems playing on your site

### ISSUE THREAD

**Issue by mpotoc** [gorgos/RouletteContract#3]
is your site at https://roulette.netlify.com/ not working, because it does not allow me to play nor does it fetch my metamask rinkeby balance? Thank you

**Comment by gorgos** [comment]
That is correct. There are currently two problems that I am aware of. Firstly, the new [MetaMask privacy mode](https://medium.com/metamask/privacy-mode-is-now-enabled-by-default-1c1c957f4d57) breaks the app. And secondly, the UI is terribly non-responsive.

There are also a lot of other things that could be updated.
